tests/integration_tests/tests/agentless_tests/test_events.py

```diff
@@ -231,6 +231,10 @@ def setUpClass(cls):
             "ALTER USER {} SET TIME ZONE '{}'"
             .format(postgres_conf.username, cls.TIMEZONE)
         )
+        # restart all users of the db so that they get a new session which
+        # uses the just-set timezone
+        docl.execute(
```

Comment:
without this restart, the above `ALTER USER` did absolutely nothing!!! because all users of the db just stayed on their previous sessions